src/Zinc-Resource-Meta-Core.package/ZnUrl.class/class/image.st

```diff
@@ -3,4 +3,4 @@ image
 	"Return a File URL refering to the file path of the current image file.
 	This is useful as a context to resolve relative URLs, see #withRelativeReference:"
 	
-	^ SmalltalkImage current imagePath asPath asZnUrl
\ No newline at end of file
+	^ (FileSystem disk resolve: SmalltalkImage current imagePath) asZnUrl
```

Comment:
Why is it needed to resolve the path in the disk?

With the new `asPath` implementation, the following yields an absolute path:

```smalltalk
SmalltalkImage current imagePath asPath
```

In OSX, this works right:

```smalltalk
SmalltalkImage current imagePath asPath asZnUrl asFileReference exists
```

Maybe there is a problem in windows?